source/_components/device_tracker.fortios.markdown

```diff
@@ -0,0 +1,33 @@
+
+This component enables Home Assistant to do device tracking of devices with a MAC address connected to a FortiGate from [Fortinet](https://www.fortinet.com).
+
+The component rely on fortiosapi: https://pypi.org/project/fortiosapi/
+The component has been tested both on FortiGate appliance and FortiGate VM running SW FortiOS v. 6.0.x and 6.2.0.
+
+All devices with a MAC address identified by FortiGate would be tracked, this covers both Ethernet and WiFi devices, incl. devices detected by LLDP.
+
+The component is based on the Home Assistant `device_tracker` platform.
+
+```yaml
+# Example configuration.yaml entry
+device_tracker:
+  - platform: fortios
+    host: 192.168.0.1
+    token: YOUR_API_USER_KEY
+    verify_ssl: False
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.